TESTS/mbed_platform/stats_cpu/main.cpp

```diff
@@ -29,7 +29,12 @@ using namespace utest::v1;
 
 DigitalOut led1(LED1);
 
+#if defined(__CORTEX_A9) || defined(__CORTEX_M23) || defined(__CORTEX_M33) || defined(__CORTEX_M7)
+#define MAX_THREAD_STACK        512
+#else
```

Comment:
I don't see the link between core and memory size....?

Reply:
Targets with these cores have their RAM enough size.

Reply:
Can you add a comment above stating this assumption.

Reply:
Done